### PR TITLE
feat(zk_toolbox): deploy legacy bridge

### PR DESCRIPTION
**Pull request by Deniallugo**
## What ❔

Support legacy bridges in zk toolbox 

## Why ❔

We still have some legacy bridges in production, so we have to have an opportunity to test it 

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.

### CODE REVIEW

File: zk_toolbox/crates/zk_supervisor/src/commands/test/mod.rs [hunk 1]
```diff
@@ -43,6 +44,8 @@ pub enum TestCommands {
     Prover,
     #[clap(about = MSG_TEST_WALLETS_INFO)]
     Wallet,
+    #[clap(about = MSG_TEST_WALLETS_INFO)]
```
Comment (perekopskiy) [inline zk_toolbox/crates/zk_supervisor/src/commands/test/mod.rs:47]: please change message

File: zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs [hunk 2]
```diff
@@ -17,4 +17,6 @@ pub struct RunServerArgs {
     additional_args: Vec<String>,
     #[clap(long, help = MSG_SERVER_BUILD_HELP)]
     pub build: bool,
+    #[clap(help=MSG_SERVER_URING_HELP, long, default_missing_value = "true", num_args = 0..=1)]
```
Comment (perekopskiy) [inline zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs:20]: zk tool uses false by default, do you think it should be true?

**Review by alexandrst88 — DISMISSED** [review]

File: zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs (hunk 2 above)
Comment (Deniallugo) [reply zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs:20]: here false is by default, just `--uring` makes it true

Comment (perekopskiy) [reply zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs:20]: ah, got it, thanks

File: zk_toolbox/crates/zk_supervisor/src/commands/test/mod.rs (hunk 1 above)
Comment (Deniallugo) [reply zk_toolbox/crates/zk_supervisor/src/commands/test/mod.rs]: fixed

**Review by alexandrst88 — APPROVED** [review]

**Review by perekopskiy — APPROVED** [review]

**Pull request merged**